library/Fission/User/CID/Mutation.hs

```diff
@@ -2,6 +2,8 @@
 module Fission.User.CID.Mutation
   ( create
   , createX
+  , deleteAll
+  , deleteExactUserCid
```

Comment:
Totally fine to leave this named as such. However, we can also use prefixes to disambiguate when needed.

```haskell
import Fission.User.CID.Mutation as User.CID

foo = User.CID.deleteExact userId cid
```